pom.xml

```diff
@@ -318,9 +308,9 @@
     <junit.version>4.11</junit.version>
     <google.api.version>1.22.0</google.api.version>
     <google.auth.version>0.4.0</google.auth.version>
-    <google.api.genomics.version>v1-rev79-1.22.0</google.api.genomics.version>
+    <google.api.genomics.version>v1-rev87-1.22.0</google.api.genomics.version>
     <google.guava.version>19.0</google.guava.version>
-    <io.grpc.version>0.14.0</io.grpc.version>
+    <io.grpc.version>0.14.1</io.grpc.version>
```

Comment:
Thanks for the bump, so I'll close https://github.com/googlegenomics/utils-java/issues/98 but I'll leave https://github.com/googlegenomics/dataflow-java/issues/205 as `dataflow-java` would need also be updated to pick up the `utils-java` update.